database/seeders/UsersSeeder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use App\Models\User;
+use Database\Factories\UserFactory;
+
+class UsersSeeder extends Seeder
```

Comment:
Good this is nice idea to adding factory seeder

Reply:
yap more testable if your application need unit or feature test